modules/reference/pages/internal-metrics-reference.adoc

```diff
@@ -123,6 +129,46 @@ Number of Kafka RPC service errors.
 
 ---
 
+=== vectorized_ntp_archiver_compacted_replaced_bytes
+
+Number of bytes removed from cloud storage by compaction operations. This is tracked on a per-topic and per-partition basis.
+
+This metric resets every time partition leadership changes. It tracks whether or not compaction is performing operations on cloud storage.
+
+The `namespace` label supports the following options for this metric:
+
+* `kafka` - User topics
+* `kafka_internal` - Internal Kafka topic, such as consumer groups
+* `redpanda` - Redpanda-only internal data
+
+*Labels*:
+
+* `namespace=("kafka" | "kafka_internal" | "redpanda")`
+* `topic`
+* `partition`
+
+---
+
+=== vectorized_ntp_archiver_pending
+
+The difference between the last committed offset and the last offset uploaded to Tiered Storage for each partition. A value of zero for this metric indicates that all data for a partition is uploaded to Tiered Storage.
+
+This metric is impacted by the xref:reference:tunable-properties.adoc#cloud_storage_segment_max_upload_interval_sec[`cloud_storage_segment_max_upload_interval_sec`] tunable property. If this interval is set to 10 seconds, the archiver will upload committed segments to Tiered Storage every 10 seconds or less. If this metric continues growing for longer than the configured interval, it can indicate a potential network issue with the upload path for that partition.
```

Comment:
Let's change 10 seconds to something more reasonable like 5 minutes. We don't want the user to use 10 seconds as an upload interval because it will be too expensive in cloud costs. Sorry, I used this as an example without thinking twice. But we better use 5 or 10 minutes there because the user will see this line and think that 10 sec is acceptable value for the property and then they will receive five-figure bill at the end of the month.